src/App.jsx

```diff
@@ -9,32 +10,19 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [value, setValue] = useState(tabs[0].id);
+  const { title } = value;
```

Comment:
1. it'd more descriptive to name selectedTabId
2. no needs to use destruction if you get only one value from 
```suggestion
  const [selectedTabId, setSelectedTabId] = useState(tabs[0].id);
```